src/toil_scripts/lib/urls.py

```diff
@@ -24,7 +24,9 @@ def download_url(url, work_dir='.', name=None, s3_key_path=None, cghub_key_path=
     if cghub_key_path:
         _download_with_genetorrent(url, file_path, cghub_key_path)
     elif urlparse(url).scheme == 's3':
-        _s3am_with_retry(num_cores=1, file_path=file_path, s3_url=url, mode='download', s3_key_path=s3_key_path)
+        _s3am_with_retry(num_cores=1, file_path=file_path, s3_url=url, mode='download',
+
```

Comment:
Nit: extra space.

Also, this is personal preference, but if I was going to break a call across multiple lines, I'd give each parameter (or at least, each logical group of parameters) their own line.